backend/server.js

```diff
@@ -0,0 +1,62 @@
+const cors = require('cors');
+const morgan = require('morgan');
+const express = require('express');
+const SequelizeStoreInit = require('connect-session-sequelize');
+const session = require('express-session');
+const Sequelize = require('sequelize');
```

Comment:
Sequelize is another ORM, similar to Prisma. If you choose to use Prisma here, then there is no need to include sequelize.

Reply:
To Do:
Delete sequelize db & pass content to prisma